api/src/pcapi/routes/backoffice_v3/templates/components/search/result_card.html

```diff
@@ -20,6 +20,12 @@ <h6 class="card-subtitle mb-4 text-muted">User ID : {{ row.id }}</h6>
           <span class="fw-bold">Tél :</span> {{ row.phoneNumber | empty_string_if_null }}
         </p>
       {% endif %}
+      {% if row.suspension_reason %}
```

Comment:
```suggestion
      {% set suspension_reason = row.suspension_reason %}
      {% if suspension_reason %}
```
Détail : comme la méthode `suspension_reason` fait un calcul en parcourant les données`, il peut être plus optimal de ne la récupérer qu'une fois et ensuite utiliser la variable en-dessous.
Mais ça reste anecdotique :)